app.py

```diff
@@ -80,16 +86,18 @@
 
 app_service_props = ServiceProps(
     container_name="schematic-app",
-    container_location="ghcr.io/sage-bionetworks/schematic:v0.1.94-beta",
+    container_location=environment_variables["SCHEMATIC_CONTAINER_LOCATION"],
     container_port=443,
-    container_memory=1024,
+    container_memory=4096,
     container_env_vars=telemetry_environment_variables,
     container_secrets=[
         ServiceSecret(
             secret_name=f"{stack_name_prefix}-DockerFargateStack/{environment}/ecs",
             environment_key="SECRETS_MANAGER_SECRETS",
         )
     ],
+    auto_scale_min_capacity=3,
+    auto_scale_max_capacity=5,
```

Comment:
Related configuration in the old deployment:
https://github.com/Sage-Bionetworks/schematic-infra/blob/dev/docker_fargate/docker_fargate_stack.py#L87-L88